src/components/Feed/Socialize/FeedEventSocializeSection.tsx

```diff
@@ -0,0 +1,267 @@
+import { useFragment } from 'react-relay';
```

Comment:
this file captures this entire section 
<img width="868" alt="image" src="https://user-images.githubusercontent.com/6754223/195633701-0f456caf-81ad-4990-9591-ec79fd4dbd25.png">